vhd/packer/configure-windows-vhd.ps1

```diff
@@ -195,9 +195,9 @@ function Install-WindowsPatches {
             # Windows Server 2019 update history can be found at https://support.microsoft.com/en-us/help/4464619
             # then you can get download links by searching for specific KBs at http://www.catalog.update.microsoft.com/home.aspx
 
-            # KB4598230 contains January 12, 2021 cumulative updates for Windows Server 2019
-            # https://www.catalog.update.microsoft.com/Search.aspx?q=KB4598230
-            $patchUrls = @("http://download.windowsupdate.com/d/msdownload/update/software/secu/2021/02/windows10.0-kb4601345-x64_6dfee9d6f028678d7988eb35cd5c0867bf96e4c6.msu")
+            # KB4598230 contains March 9, 2021 cumulative updates for Windows Server 2019

```

Comment:
Do you need to update the base image to 17763.1757.2102060435?

Reply:
I was going to add this comment too.
I don't think it *needs* to be done but would be nice to have.

Reply:
done